src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -177,6 +178,7 @@ def __init__(self, case_filepath: str = None, project_filepath: str = None):
                 case_filepath = Path(
                     project_dir + _get_case_filepath_from_flprj(project_filepath)
                 )
+                a = 1
```

Comment:
accident? @prmukherj

Reply:
Thank you @seanpearsonuk, had added it for debugging.

Reply:
yes, true. We will raise a FileNotFoundError.
Thank you.

Reply:
https://stackoverflow.com/a/29442282

Reply:
Done, Thank you.